internal/painter/gl/gl_common.go

```diff
@@ -131,7 +131,7 @@ func (p *glPainter) newGlImageTexture(obj fyne.CanvasObject) Texture {
 		return NoTexture
 	}
 
-	return p.imgToTexture(tex)
+	return p.imgToTexture(tex, img.ScalingFilter)
```

Comment:
We should sanity check this value I think - if the user sets it to "2" we will crash with index out of bounds in the draw code...